src/main/java/com/sourcegraph/cody/SubscriptionTabUtils.kt

```diff
@@ -0,0 +1,81 @@
+package com.sourcegraph.cody
+
+import com.intellij.openapi.project.Project
+import com.intellij.util.concurrency.annotations.RequiresBackgroundThread
+import com.sourcegraph.cody.agent.CodyAgentServer
+import com.sourcegraph.cody.agent.protocol.GetFeatureFlag
+import com.sourcegraph.cody.config.CodyAuthenticationManager
+import java.util.concurrent.CompletableFuture
+
+data class SubscriptionTabPanelData(
+    val isDotcomAccount: Boolean,
+    val codyProFeatureFlag: Boolean,
+    val isCurrentUserPro: Boolean?
+)
+
+@RequiresBackgroundThread
+fun fetchSubscriptionPanelData(
+    project: Project,
+    server: CodyAgentServer
+): CompletableFuture<SubscriptionTabPanelData?> {
+  val activeAccountType = CodyAuthenticationManager.instance.getActiveAccount(project)
+  if (activeAccountType != null) {
+    ensureUserIdMatchInAgent(activeAccountType.id, server)
+    return if (activeAccountType.isDotcomAccount()) {
+      val codyProFeatureFlag = server.evaluateFeatureFlag(GetFeatureFlag("CodyProJetBrains"))
```

Comment:
Is that doing something else than `  @JsonRequest("graphql/currentUserIsPro") fun isCurrentUserPro(): CompletableFuture<Boolean?>` method we are using across the codebase?

Reply:
Yes, this is feature flag that was used to disable/enable whole PLG (subscription tab) feature.